Sources/Private/CoreAnimation/CoreAnimationLayer.swift

```diff
@@ -19,6 +19,7 @@ final class CoreAnimationLayer: BaseAnimationLayer {
     imageProvider: AnimationImageProvider,
     textProvider: AnimationTextProvider,
     fontProvider: AnimationFontProvider,
+    maskAnimationToBounds: Bool,
```

Comment:
It shouldn't actually be necessary to pass this in since the default value is the same as what we set it to in the init. Not sure what's preferred

Reply:
I would probably prefer a design like:


```swift
let layer = CoreAnimationLayer(...)
layer.masksToBounds = maskAnimationToBounds
```

since there's no need to inject a property into the initializer when we can just configure the same thing directly at the call site. But either is fine.

Reply:
What you have here may be better anyway so we don't forget to configure this if we ever add another `init` call to either of these layer type

Reply:
This makes more sense, I'll change it

Reply:
Oh oops didn't see your second message